src/scripts/main.js

```diff
@@ -1,3 +1,267 @@
 'use strict';
 
-// write your code here
+const theOnlyButton = document.querySelector('.button');
```

Comment:
It's a bad name for a variable. What does it button represent? What is its functionality? This name doesn't give me such information at all.